### DISCUSSION TITLE
Documentation site is offline

### DISCUSSION THREAD

**Discussion by k-dahl** [stephenmcd/mezzanine#2029] [General]
Hey - the documentation site seems to be offline since sometime mid last week. Just a heads up - http://mezzanine.jupo.org/sites/

@jerivas (not sure who else to tag)

**Comment by HunterGrey-cyber** [comment]
what happened

**Comment by jerivas** [comment]
Thanks for reporting this problem. I have opened https://github.com/stephenmcd/mezzanine/issues/2030 to track this. Contributions are welcome.
